localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -374,6 +419,24 @@
         <trans-unit id="msgConnect">
             <source xml:lang="en">Connect</source>
         </trans-unit>
+        <trans-unit id="azureSignIn">
+            <source xml:lang="en">Azure: Sign In</source>
```

Comment:
@kburtram  I copied the `Command Palette options` from the account extension to show all the commands at once since each option has a different command. It looks something like this -

![Capture](https://user-images.githubusercontent.com/6411451/69755256-aee8e600-110c-11ea-97f0-33835b9426fa.JPG)

I can change `Azure` to `MSSQL` if we want.